README.md

```diff
@@ -20,8 +20,16 @@ The above example will output coverage metrics for `foo.js`.
 
 run `c8 report` to regenerate reports after `c8` has already been run.
 
-## Disclaimer
+## Supported Node.js Versions
 
 c8 uses
 [bleeding edge Node.js features](https://github.com/nodejs/node/pull/22527),
 make sure you're running Node.js `>= 10.10.0`.
+
+## Goals of Project
+
+A fully functional code coverage solution using only V8's native coverage
+features and minimal user-land modules:
```

Comment:
Being strict here, this sentence does not really tell me what the following list is for. I would add:

`...modules, so that we fit these constraints:` or something to that effect.

Reply:
Glad to make this change 👍